### DISCUSSION TITLE
[Question][Hw2] Semantics of `other instructions` seems to clash.

### DISCUSSION THREAD

**Discussion by Lee-Janggun** [prosyslab-classroom/cs524-program-analysis#419] [Q&A]
I think there is a clash of semantics for instructions of unsupported instructions in HW2, namely the following two.

* The last line of `# SmaLLVM Interpreter ` says that "You can ignore the other LLVM instructions marked as `raise Unsupported`,..."
* The last line of `4. Format of Input Programs` says that `All the other instructions are considered to be nop.`. 

From the first description, it seems that we should leave the `raise Unsupported` as is. The second description makes it seems that we should treat such instructions as `Skip` as we have seen in class, and I was unsure which one we should take.

I assume that due to the constraint that "input programs are always **syntactically** valid.", we really don't have to worry about such clash during tests, but I still wanted to ask as I was wondering what would be the better choice in other situations.

Thanks in advance!

**Comment by KihongHeo** [comment]
You can ignore unsupported syntax.
